components/automate-ui/src/app/modules/policy/details/policy-details.component.ts

```diff
@@ -103,11 +107,13 @@ export class PolicyDetailsComponent implements OnInit, OnDestroy {
     return JSON.stringify(policy);
   }
 
-  removeMember(member: Member): void {
-    this.store.dispatch(new RemovePolicyMembers(<PolicyMembersMgmtPayload>{
-      id: this.policy.id,
-      members: [member]
-    }));
+  removeMember($event: KeyboardEvent, member: Member): void {
+    if ($event.isUserInput) {
+      this.store.dispatch(new RemovePolicyMembers(<PolicyMembersMgmtPayload>{
+        id: this.policy.id,
+        members: [member]
+      }));
+    }
```

Comment:
An example of using the `$event.isUserInput` to only get the selection and not the deselection.